addons/hr_timesheet/report/project_report.py

```diff
@@ -14,16 +14,16 @@ class ReportProjectTaskUser(models.Model):
 
     def _select(self):
         return super(ReportProjectTaskUser, self)._select() + """,
-            (t.effective_hours * 100) / NULLIF(planned_hours, 0) as progress,
+            (t.effective_hours * 100) / NULLIF(t.planned_hours, 0) as progress,
             t.effective_hours as hours_effective,
             t.planned_hours - t.effective_hours - t.subtask_effective_hours as remaining_hours,
-            NULLIF(planned_hours, 0) as hours_planned"""
+            NULLIF(t.planned_hours, 0) as hours_planned"""
 
     def _group_by(self):
         return super(ReportProjectTaskUser, self)._group_by() + """,
-            remaining_hours,
+            t.remaining_hours,
```

Comment:
Is this alias correct/needed?
It looks like a column created for the select result, thus I wouldn't take the one from the base table (`project_task t`)
https://github.com/odoo/odoo/blob/4d11cb0ee7f305550496e90327afd0376f9835d6/addons/project/report/project_report.py#L97-L103
it may read the wrong info
https://github.com/odoo/odoo/blob/4d11cb0ee7f305550496e90327afd0376f9835d6/addons/hr_timesheet/models/project.py#L240
Although it seems like the compute does precisely the same? :/
https://github.com/odoo/odoo/blob/4d11cb0ee7f305550496e90327afd0376f9835d6/addons/hr_timesheet/models/project.py#L287

Reply:
> The field is stored, so in the table.

Indeed. I wonder why we recompute it then?
https://github.com/odoo/odoo/blob/48f178fc348148b5548060cd31c2d80f5c226a9f/addons/hr_timesheet/report/project_report.py#L19
`t.planned_hours - t.effective_hours - t.subtask_effective_hours as remaining_hours,` -> `t.remaining_hours as remaining_hours`?

Reply:
¯\\_(ツ)_/¯